pkg/kube/kube_handler_test.go

```diff
@@ -530,9 +530,23 @@ func TestDeleteAppRepository(t *testing.T) {
 				clientsetForConfig: func(*rest.Config) (combinedClientsetInterface, error) { return cs, nil },
 				kubeappsNamespace:  kubeappsNamespace,
 				svcClientset:       cs,
+				clustersConfig: ClustersConfig{
+					KubeappsClusterName: "cluster",
+					Clusters: map[string]ClusterConfig{
+						"cluster": {
+							Name:                     "cluster",
+							APIServiceURL:            "fake",
+							CertificateAuthorityData: "fake",
+							CAFile:                   "",
+							ServiceToken:             "fake",
+							Insecure:                 true,
+						},
+					},
+				},
 			}
 
-			err := handler.AsSVC().DeleteAppRepository(tc.repoName, tc.requestNamespace)
+			cli, _ := handler.AsSVC("cluster")
```

Comment:
Just pls handle the error for completeness. I know it seems unnecessary but I did recently get a nil pointer error the other week only to find it was because of an unhandled error.